teamworkApp/lib/studentGraph.py

```diff
@@ -1,17 +1,44 @@
 import argparse 
 import matplotlib.pyplot as plt
+import numpy as np
+from enum import IntEnum
+import sqlite3
 
-from evaluateAnswers import * 
+DB = 'db/development.sqlite3'
+Style = IntEnum('Style', 'Contributor, Collaborator, Communicator, Challenger', start=0)
 
 def student_graph(student_id):
     """Returns a path to where the student's graph is saved"""
 
     labels = [Style(x).name for x in range(len(Style))]
-    scores = find_scores(student_id)
 
-    fig1, ax1 = plt.subplots()
-    ax1.pie(scores, labels=labels, autopct='%1.1f%%', shadow=True, startangle=90)
-    ax1.axis('equal')
+    # Connect to db to get scores
+    conn = sqlite3.connect(DB)
+    c = conn.cursor()
+    c.execute(
+        'SELECT * FROM styles WHERE student_id=?', 
+        [student_id,],
+    )
+
+    dbData = c.fetchall()
+
+    scores = [0] * 4 
+    if len(dbData) == 0:
+        print("Error: Could not find any data on this student")
+    else:
+        scores = dbData[0][2:6]
+    
+    conn.commit()
+    conn.close()
```

Comment:
The connection can be closed after line 23 of this file which I think would be cleaner.